.github/workflows/image_build_and_test.yaml

```diff
@@ -69,7 +69,7 @@ jobs:
 
       - name: Extract branch name
         shell: bash
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF_NAME})"
```

Comment:
would `GITHUB_REF#refs/*/` work? that's what we use everywhere else

Reply:
https://github.com/uc-cdis/.github/blob/ca5ac2d/.github/workflows/image_build_push.yaml#L93

Reply:
it is breaking here. It seems that there is some new change in Github. `GITHUB_REF_NAME` is the new feature

Reply:
tube is using `GITHUB_REF#refs/heads/` and i'm suggesting `GITHUB_REF#refs/*/` slightly different since it handles tags too
if `GITHUB_REF#refs/*/` does not work anymore, the builds would be broken for a lot of other repos

Reply:
but `GITHUB_REF_NAME` is the new feature and we do not rely on the regex anymore. I think it is better @paulineribeyre

Reply:
ok yes that's fine
i'm just trying to see if our other repos are going to break too